src/components/chartRepo/ChartRepo.tsx

```diff
@@ -89,9 +89,19 @@ function CollapsedList({ id, name, active, url, authMode, accessToken = "", user
     }, [enabled])
 
     const setToggleCollapse = () => {
-        toggleCollapse(false)
+        toggleCollapse(true)
     }
 
+    const handleCollapse = (e) => {
+        if (isEditable) {
+            e.stopPropagation();
+            toggleCollapse(t => !t)
+        } else {
+            const msg = "Cannot edit chart repo \""+name+"\"."+
```

Comment:
this can be written as toast.info(`Cannot edit chart repo "${name}". Some charts from this repository are being used by helm apps.`)